packages/www/pages/docs/index.tsx

```diff
@@ -15,31 +15,31 @@ const categories: React.ComponentProps<typeof DocsCategoryCard>[] = [
     img: {
       src: "/img/docs/flag.png",
       alt: "flag",
-      sx: { width: "140px" }
+      sx: { width: "105px" }
     },
-    title: "Getting started",
-    description: "Start live streaming now!",
+    title: "Start with Livepeer.com guides",
+    //description: "Start live streaming now!",
     link: { href: "/docs/guides" }
   },
   {
     img: {
       src: "/img/docs/puzzle.png",
       alt: "puzzle",
-      sx: { width: "140px" }
+      sx: { width: "105px" }
     },
-    title: "Feature support matrix",
-    description: "Learn about compatible video formats and workflows.",
+    title: "Supported codecs & workflows",
+    //description: "Learn about compatible video formats and workflows.",
     link: { href: "/docs/guides/support-matrix" }
   },
   {
     img: {
       src: "/img/docs/stopwatch.png",
       alt: "stopwatch",
-      sx: { width: "120px" }
+      sx: { width: "90px" }
     },
-    title: "Your first stream in under 5 minutes",
-    description: "Read the Livepeer.com blog post.",
-    link: { href: "/blog/first-livepeer-stream-in-five-minutes" }
+    title: "Debugging Stream Playback",
+    //description: "Solve common stream playback problems.",
```

Comment:
Hey @wohlner - the missing description property is what's causing this deployment to fail. It's a required field in this component.

Reply:
I don't want to show the description any more. I think I successfully removed the requirement from the component. Thank you!